distribution/src/release/LICENSE

```diff
@@ -721,3 +721,9 @@ For details, see https://github.com/webjars
 * SockJS-client (org.webjars:sockjs-client:1.0.2 - http://webjars.org)
 * webjars-locator (org.webjars:webjars-locator:0.36 - http://webjars.org)
 * webjars-locator-core (org.webjars:webjars-locator-core:0.35 - http://webjars.org)
+
+===========================================================================
```

Comment:
As we include the script in the source bundle, we need to update the License in the root directory.